media/doc/WINESYNC.txt

```diff
@@ -325,13 +325,13 @@ regedit
   base/applications/regedit/regproc.c   # Synced to Wine-7.17
 
 user32 -
-  win32ss/user/user32/controls/button.c    # Synced to WineStaging-1.7.37
-  win32ss/user/user32/controls/combo.c     # Synced to WineStaging-1.7.37
-  win32ss/user/user32/controls/edit.c      # Synced to WineStaging-1.7.55
-  win32ss/user/user32/controls/icontitle.c # Synced to WineStaging-1.7.55
-  win32ss/user/user32/controls/listbox.c   # Synced to WineStaging-1.7.55
+  win32ss/user/user32/controls/button.c    # Synced to Wine-4.18
+  win32ss/user/user32/controls/combo.c     # Synced to Wine-4.18
+  win32ss/user/user32/controls/edit.c      # Synced to Wine-4.18
+  win32ss/user/user32/controls/icontitle.c # Synced to Wine-4.18
+  win32ss/user/user32/controls/listbox.c   # Synced to Wine-4.18
   win32ss/user/user32/controls/scrollbar.c # Forked
-  win32ss/user/user32/controls/static.c    # Synced to WineStaging-1.7.55
+  win32ss/user/user32/controls/static.c    # Synced to Wine-4.18
```

Comment:
Have you checked whether WineStaging 4.18 has extra fixes for user32 controls?

Reply:
Nothing useful, and I’m Likely going to synch higher I think I haven’t decided yet